docs/upgrade/3.10.0.rst

```diff
@@ -67,8 +67,8 @@ For more information, please see: https://github.com/django-cms/django-cms/pull/
 
 Then run::
 
-    python manage.py migrate  # to ensure that your database is up-to-date with migrations
-    python manage.py cms fix-tree
+    ``python manage.py migrate``  # to ensure that your database is up-to-date with migrations
```

Comment:
I don't think the back-ticks are necessary here, because the `::` ending the previous paragraph turns lines 70 & 71 into pre-formatted text. See here; https://docutils.sourceforge.io/docs/user/rst/quickstart.html#preformatting-code-samples

Reply:
Hi Mark, 

Thank you for catching this error. I got rid of the backticks, you were right. I will make a pull request soon with this change and other ones. 

Best, 
Luis

Reply:
@luisnavarrodelangel You can still push changes to the branch for this PR to address this, and anything else. That works better than merging this, when there may be comments to address, then submitting another PR to address.